include/onnxruntime/core/session/onnxruntime_cxx_inline.h

```diff
@@ -1555,7 +1662,7 @@ inline const OrtMemoryInfo* CustomOpApi::GetTensorMemoryInfo(_In_ const OrtValue
 
 template <typename T>
 inline const T* CustomOpApi::GetTensorData(_Inout_ const OrtValue* value) {
-  return GetTensorData<T>(value);
+  return GetTensorMutableData<T>(const_cast<OrtValue*>(value));
```

Comment:
Fix infinite recursion bug introduced in https://github.com/microsoft/onnxruntime/commit/5dae0c477d6c70d91952fb2c11fad37301cde7ae

Reply:
PR: https://github.com/microsoft/onnxruntime/pull/13450